types/jss/index.d.ts

```diff
@@ -3,118 +3,305 @@
 // Definitions by: Brenton Simpson <https://github.com/appsforartists>
 //                 Oleg Slobodskoi <https://github.com/kof>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.3
 
-export interface Rule {
-	className: string;
-	selector: string;
-	applyTo(element: HTMLElement): void;
-	prop(key: string): string;
-	prop(key: string, value: any): this;
-	toJSON(): string;
+import { CSSProperties } from 'react';
```

Comment:
@meyer uses `CSSProperties` in `jsxstyle` also.  Makes me wonder if they should be broken out into their own typings, e.g. `@types/css`.  (We'd have to figure out where to put them.  `css`, `css-properties`, and `typed-css` are all already taken on NPM.)

https://github.com/smyte/jsxstyle/blob/master/misc/mktypes.js

Reply:
@appsforartists re where to put them, a scoped package would be a good way to go IMO:`@cssinjs/packagename`.

Reply:
Oohhh, `@types/cssinjs` would work too.